editoast/src/schema/buffer_stop.rs

```diff
@@ -1,16 +1,23 @@
-use crate::infra_cache::Cache;
-use crate::infra_cache::ObjectCache;
+use crate::api_error::ApiError;
+
+use crate::infra_cache::{Cache, ObjectCache};
 use crate::layer::Layer;
 
 use super::generate_id;
 use super::ApplicableDirections;
 use super::OSRDObject;
 use super::ObjectType;
 use derivative::Derivative;
-use diesel::sql_types::{Double, Text};
+
 use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+use diesel::pg::types::sql_types::Json;
+use diesel::sql_types::{Double, Integer, Text};
+use diesel::ExpressionMethods;
+use diesel::{PgConnection, QueryDsl, RunQueryDsl};
 
-#[derive(Debug, Derivative, Clone, Deserialize, Serialize)]
+#[derive(Debug, Derivative, Clone, Deserialize, Serialize, Queryable)]
```

Comment:
Not used

Reply:
well seen ! was added in the wrong struct at first